pkg/pillar/hypervisor/kvm.go

```diff
@@ -452,27 +486,7 @@ func (ctx kvmContext) CreateDomConfig(domainName string, config types.DomainConf
 		PCIId, DiskID, SATAId int
 		AioType               string
 		types.DiskStatus
-	}{Machine: ctx.devicemodel, PCIId: 4, DiskID: 0, SATAId: 0, AioType: "threads"}
-
-	var osver []string
-	var major, minor, patch uint64
-
-	osver = strings.SplitN(getOsVersion(), ".", 3)
```

Comment:
We can discuss separately whether dropping this is a good idea -- but please lets make it part of a separate PR @temerkhanov

Reply:
Moved to another PR

Reply:
Moved to a different PR